src/lib/zAuction.ts

```diff
@@ -57,29 +48,18 @@ interface BidPostInterface {
 	signedMessage: string;
 }
 
-interface BidAcceptInterface {
-	signedMessage: string;
-	auctionId: string;
-	account: string;
-	bidAmount: string;
-	contractAddress: string;
-	tokenId: string;
-	minimumBid: string;
-	startBlock: string;
-	expireBlock: string;
-}
-
-function getApiEndpoint() {
-	// const chainSelector = useChainSelector();
-	let chain = 42; //useChainSelector().selectedChain;
-	switch (chain) {
-		case 1:
-			return 'https://zproduction.ilios.dev/api';
-		case 42:
-			return 'https://zproxy.ilios.dev/api';
-		default:
-			return 'Unsupported Chain';
-	}
+function setApiEndpoints(apiName: string) {
```

Comment:
Nit: rename to `getApiEndpoints`

A function with `set` in it's name implies you are setting the state of some long-lived object. This function does not write to any state, it returns a newly created object. "Get" is a more proper verb for this sort of action.